src/Domain/User/Exception/SiretAlreadyRequestedException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Domain\User\Exception;
+
+final class SiretAlreadyRequestedException extends \Exception
```

Comment:
(Détail) Naming : requested -> exists ?

```suggestion
final class SiretAlreadyExistsException extends \Exception
```